src/reaper_osara.cpp

```diff
@@ -1595,6 +1595,26 @@ void postCopy(int command) {
 	}
 }
 
+void postCopyMoveTimeSelection(int command) {
+	ostringstream s;
+	double start, end;
+	GetSet_LoopTimeRange(false, true, &start, &end, false);
+	fakeFocus = FOCUS_RULER;
+	if (start != end && command == 40397) {
```

Comment:
This can be made a bit cleaner. You can check if start == end, then outputMessage no time selection and return. That avoids the repeated start != end checks and also the s.tellp check.